app/src/main/kotlin/com/fernandocejas/sample/features/movies/MovieDetailsFragment.kt

```diff
@@ -45,9 +47,10 @@ class MovieDetailsFragment : BaseFragment() {
         }
     }
 
-    @Inject lateinit var movieDetailsAnimator: MovieDetailsAnimator
+    @Inject
```

Comment:
Please keep the old style in one line if possible. This is for consistency. I'm happy to change the style if that is official but then for consistency we have to change it in the whole codebase.  :)